crates/stc_ts_file_analyzer/tests/pass-only/conformance/types/tpl/stringMappingOverPatternLiterals/1.ts

```diff
@@ -0,0 +1,15 @@
+// non-template
+type A = "aA";
+type B = Uppercase<A>;
+type C = Lowercase<A>;
+
+// templated
+type ATemplate = `aA${string}`;
+type BTemplate = Uppercase<ATemplate>;
+type CTemplate = Lowercase<ATemplate>;
+
+let aTemp: ATemplate = "aAFoo";
+let bTemp: BTemplate = "AAFOO";
+let cTemp: CTemplate = "aafoo";
+
+export {};
```

Comment:
Moved to `pass/*` as none of the cases failed.

Reply:
Please move it back to `pass/`. 
`pass-only` does not dump types

Reply:
If `std-err` doesn't print, should I upload an empty file?

Reply:
Yes, but no file is also fine